### DISCUSSION TITLE
Top 3 MUST have standards?

### DISCUSSION THREAD

**Discussion by loub-lu** [T4RJ/community#2] [Security]
Hi all,
**Upvote your top three must have standards!** If you have others to add, feel free to toss them in this list as new comments. If you have questions or clarifications about these, reply to the individual comment.

**Comment by loub-lu** [comment]
Lock user accounts after 3 months inactivity

**Comment by loub-lu** [comment]
Force 2FA

**Comment by loub-lu** [comment]
Data retention (regulations) + have a plan in place//data retention policy

**Comment by loub-lu** [comment]
Encryption at rest (field level for extra sensitive)

**Comment by loub-lu** [comment]
Require two people to do high risk things

**Comment by loub-lu** [comment]
Access/visibility to logging

**Comment by loub-lu** [comment]
Production ready checklist

**Comment by loub-lu** [comment]
Sensitive info redacting in logs (reply with how tos!)

**Comment by loub-lu** [comment]
Regular maintenance + security testing schedule

**Comment by loub-lu** [comment]
Access audits (who has it)

**Comment by loub-lu** [comment]
Principle of least authority (explain below)

**Reply by loub-lu** [reply]
people should have only the access that they absolutely need

**Comment by loub-lu** [comment]
MFA as an option (enforceable at org level)

**Comment by loub-lu** [comment]
Trusted authentication framework

**Reply by loub-lu** [reply]
for all sign in and sign off, make sure the library/framework is trusted/in active development; something that is tested and good

**Comment by loub-lu** [comment]
IP logs

**Comment by loub-lu** [comment]
Password reset

**Comment by loub-lu** [comment]
Strong account roles

**Comment by loub-lu** [comment]
Keeping friction where it makes sense (good friction)

**Comment by loub-lu** [comment]
Data access controls

**Comment by loub-lu** [comment]
Alerting/bug reporting

**Comment by loub-lu** [comment]
Written access/privilege protocol

**Comment by loub-lu** [comment]
Separation of tenant data

**Reply by d-m-wilson** [reply]
I think this means _tenant_ data (for multitenant systems). Is that right?

**Reply by loub-lu** [reply]
probably! thanks for catching that

**Comment by loub-lu** [comment]
Automated testing

**Comment by loub-lu** [comment]
Escalation path

**Reply by engineear1** [reply]
What does this one mean?
